test/unit/controllers/declaration/DeclarationAdditionalActorsControllerSpec.scala

```diff
@@ -164,6 +164,17 @@ class DeclarationAdditionalActorsControllerSpec extends ControllerSpec with Erro
 
     "return 303 (SEE_OTHER)" when {
 
+      onJourney(CLEARANCE)() { request =>
+        "redirect 303 (See Other) to start" in new SetUp {
```

Comment:
Wording in this test looks little silly
``` return 303 (SEE_OTHER) when on clearance journey redirect 303 (See Other) to Start``` - is here you mean "back to journey start" ?